scripts/tiling/stitching.py

```diff
@@ -0,0 +1,256 @@
+
+            full_id = exists['results'][0]['id'] if exists['count'] else None
+
+            if not full_id:
+                new_im_metadata = {}
+                new_im_metadata['run'] = ref_json['run']['id']
+                new_im_metadata['point'] = sample_id
+                new_im_metadata['number'] = out_mibi_tiff.fov_id
+                new_im_metadata['folder'] = out_mibi_tiff.folder
+                new_im_metadata['fov_size'] = max_dim/px_per_u
+                # assumption that all fovs are same
+                new_im_metadata['dwell_time'] = ref_json['dwell_time']
+                # assumption that all fovs are same
+                new_im_metadata['depths'] = ref_json['depths']
+                new_im_metadata['frame'] = max_dim
+                # assumption that all fovs are same
+                new_im_metadata['time_bin'] = ref_json['time_bin']
+                # assumption that all fovs are same
+                new_im_metadata['mass_gain'] = ref_json['mass_gain']
+                # assumption that all fovs are same
+                new_im_metadata['mass_offset'] = ref_json['mass_offset']
+                new_im_metadata['x_coord'] = int(np.round(um_min_x))
+                new_im_metadata['y_coord'] = int(np.round(um_min_y))
+                new_im_metadata['tissue'] = \
+                    ref_json['tissue'] and ref_json['tissue']['id']
+                new_im_metadata['section'] = ref_json['section']['id']
+                # assumption that all fovs are same
+                new_im_metadata['aperture'] = ref_json['aperture']['id']
+                # assumption that all fovs are same
+                new_im_metadata['imaging_preset'] = ref_json['imaging_preset']
+                # assumption that all fovs are same
+                new_im_metadata['lens1_voltage'] = ref_json['lens1_voltage']
+
+                for t in range(MAX_TRIES):
+                    try:
+                        _ = mr.post('/images/', json=new_im_metadata)
+                        break
+                    except:
+                        if t < MAX_TRIES-1:
+                            time.sleep(0.50)
+                        else:
+                            _ = mr.post('/images/', json=new_im_metadata)
+
+            for t in range(MAX_TRIES):
+                try:
+                    mr.upload_mibitiff(out_path, run_id=ref_json['run']['id'])
+                    break
+                except:
+                    if t < MAX_TRIES-1:
+                        time.sleep(0.50)
+                    else:
+                        mr.upload_mibitiff(out_path, run_id=ref_json['run']['id'])
+            
+            print(f"Stitched MIBItiff, {out_path} uploaded to MIBItracker.")
+                        
+
+if __name__ == "__main__":
+
+    fovs_folder = "/Users/mnagy/projects/stitch_script_test/2024-04-29T10-07-46_gold_tonsil_3x3_coarse_ROI"
```

Comment:
You should remove paths to your local machine. Generally, if there are variables that a user needs to set in order for the script to work, it should be indicated and some documentation included. For example: `fovs_folder = '' # Set this to the path of the folder that contains the FOVs to stich`